rts/Lua/LuaSyncedCtrl.h

```diff
@@ -5,21 +5,23 @@
 
 struct lua_State;
 
+#define MAX_CMD_RECURSION_DEPTH 16
+
 class LuaSyncedCtrl
 {
 	public:
 		static bool PushEntries(lua_State* L);
 		static void CheckAllowGameChanges(lua_State* L);
 
 	private:
-		static bool inCreateUnit;
-		static bool inDestroyUnit;
-		static bool inTransferUnit;
-		static bool inCreateFeature;
-		static bool inDestroyFeature;
-		static bool inGiveOrder;
-		static bool inHeightMap;
-		static bool inSmoothMesh;
+		static int inCreateUnit;
```

Comment:
While you are on it please convert from `static int` into `inline static int` it's more modern and compact.
Eg. `inline static int inDestroyUnit = 0;`